packages/esm-patient-common-lib/src/form-entry-interop.ts

```diff
@@ -24,29 +22,18 @@ export function launchFormEntryOrHtmlForms(
 ) {
   if (visitUuid) {
     const htmlForm = htmlFormEntryForms.find((form) => form.formUuid === formUuid);
-    if (isEmpty(htmlForm)) {
-      launchFormEntry(
-        formUuid,
-        patientUuid,
-        encounterUuid,
-        formName,
-        visitUuid,
-        visitTypeUuid,
-        visitStartDatetime,
-        visitStopDatetime,
-        mutateForms,
-      );
-    } else {
-      if (encounterUuid) {
-        navigate({
-          to: `\${openmrsBase}/htmlformentryui/htmlform/${htmlForm.formEditUiPage}.page?patientId=${patientUuid}&visitId=${visitUuid}&encounterId=${encounterUuid}&definitionUiResource=${htmlForm.formUiResource}&returnUrl=${window.location.href}`,
-        });
-      } else {
-        navigate({
-          to: `\${openmrsBase}/htmlformentryui/htmlform/${htmlForm.formUiPage}.page?patientId=${patientUuid}&visitId=${visitUuid}&definitionUiResource=${htmlForm.formUiResource}&returnUrl=${window.location.href}`,
-        });
-      }
```

Comment:
Previously, if this form had been configured as an HTML Form, we simply redirected to the O2 app,